src/components/ProfileInput.jsx

```diff
@@ -0,0 +1,109 @@
+import React from 'react';
+import { makeStyles } from '@material-ui/core/styles';
+import InputLabel from '@material-ui/core/InputLabel';
+import MenuItem from '@material-ui/core/MenuItem';
+import FormControl from '@material-ui/core/FormControl';
+import Select from '@material-ui/core/Select';
+import Radio from '@material-ui/core/Radio';
+import RadioGroup from '@material-ui/core/RadioGroup';
+import FormControlLabel from '@material-ui/core/FormControlLabel';
+import FormLabel from '@material-ui/core/FormLabel';
+
+const useStyles = makeStyles((theme) => ({
+  formControl: {
+    margin: theme.spacing(1),
+    minWidth: 300,
+  },
+  selectEmpty: {
+    marginTop: theme.spacing(2),
+  },
+}));
+
+export default function ProfileInput() {
+  const classes = useStyles();
+  const [grade, setGrade] = React.useState('');
+  const [gradYear, setGradeYear] = React.useState('');
+  const [major, setMajor] = React.useState('');
+  const [gender, setGender] = React.useState('');
+  const [daca, setDaca] = React.useState('');
+
+  const handleGradeChange = (event) => {
+    setGrade(event.target.value);
+  };
+  const handleGradChange = (event) => {
+    setGradeYear(event.target.value);
+  };
+  const handleMajorChange = (event) => {
+    setMajor(event.target.value);
+  };
+  const handleGenderChange = (event) => {
+    setGender(event.target.value);
+  };
+  const handleDacaChange = (event) => {
+    setDaca(event.target.value);
+  };
+
+  return (
+    <div>
+      <FormControl className={classes.formControl}>
+        <InputLabel>Grade</InputLabel>
+        <Select labelId="demo-simple-select-label" id="demo-simple-select" value={grade} onChange={handleGradeChange}>
+          <MenuItem value={20}>Highschool Senior</MenuItem>
+          <MenuItem value={30}>College Freshman</MenuItem>
+          <MenuItem value={40}>College Sophomore</MenuItem>
+          <MenuItem value={50}>College Junior</MenuItem>
+          <MenuItem value={60}>College Senior</MenuItem>
+        </Select>
+      </FormControl>
+      <br />
+      <FormControl className={classes.formControl}>
+        <InputLabel>Graduation Year</InputLabel>
+        <Select
+          labelId="demo-simple-select-label"
+          id="demo-simple-select"
+          value={gradYear}
+          onChange={handleGradChange}
+        >
+          <MenuItem value={10}>2020</MenuItem>
+          <MenuItem value={20}>2021</MenuItem>
+          <MenuItem value={30}>2022</MenuItem>
+        </Select>
+      </FormControl>
```

Comment:
There's a lot of repetitive code here. I suggest refactoring this out and have it take in the the input label and list of values as properties and maybe the `value` and  `onChange` as properties as well.

Reply:
Hmm I'm trying to make sense of this but Im not sure what you are referring too, is there perhaps an example you can share for me to comprehend what it is you mean? Thank would be really helpful im going to do a quick Google search and see if I can discover it on my own as well